R/AbstractPackageReporter.R

```diff
@@ -72,18 +82,31 @@ AbstractPackageReporter <- R6::R6Class(
         get_package = function(){
             return(private$packageName)
         },
-        
+      
         get_package_path = function(){
           return(private$packagePath)
         },
-        
-        get_report =  function() {
-            stop("get_report has not been implemented.")
+   
+        get_report =  function(output_file = NULL) {
+            rmarkdown::render(self$get_report_markdown_path()
+                              , output_format = "html_document"
```

Comment:
I've seen people do this and, alternatively, use `rmarkdown::html_document()` for `output_format`. The docs make it seem like both are supported but I don't know the implications of choosing one over the other...do you?